lib/sidekiq_adhoc_job/strategies/rails.rb

```diff
@@ -0,0 +1,19 @@
+module SidekiqAdhocJob
+  module Strategies
+    class Rails
+      include SidekiqAdhocJob::Strategy
+
+      def worker_class?(klass)
```

Comment:
how about something generic that you can support both activejob and sidekiq?

```
module SidekiqAdhocJob
  module Strategies
    class Rails < Rack
      def worker_class?(klass)
        active_job?(klass) ? klass.queue_adapter.is_a?(ActiveJob::QueueAdapters::SidekiqAdapter) : super 
      end

      def get_queue_name(klass)
        active_job?(klass) ? klass.queue_name : super
      end

      private

      def active_job?(klass)
        klass.ancestors.include?(ActiveJob::Base)
      end
    end
  end
end
```

Reply:
That's a good point!

My question will then be, whether it's possible to have a mix use of Sidekiq and ActiveJob within the same Rails app?

If it's not possible, then there's no need to configure the strategy to `:rails`. It will then use the default `:rack` strategy, which checks for included modules contain `Sidekiq::Worker`.

Perhaps the fix will be to rename the strategy to `:sidekiq` and `:active_job`.

WDYT? @longkt90

Reply:
yeah,  rename the strategy to :sidekiq and :active_job is great!

Reply:
Thanks for the feedback! I will change the strategy name 😄